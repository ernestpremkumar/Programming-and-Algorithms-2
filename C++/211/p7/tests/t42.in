echo t01: Average distance of an empty tree should be 0.
distance
