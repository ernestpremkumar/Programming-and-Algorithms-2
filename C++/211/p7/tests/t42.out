t01: Average distance of an empty tree should be 0.
Average distance of nodes to root = 0
